bionic/persistence.py

```diff
@@ -645,14 +635,6 @@ def _nominal_metadata_url_prefix_for_query(self, query):
             + minor_version_token
```

Comment:
now that we have a nominal hash, how do you feel about replacing minor version with `query.provenance.nominal_hash`? That way, you won't bleed the logic of how provenance hashes work in url creation. Although, it would also mean that our urls will be even longer.

Reply:
Yeah, that seems like a good idea. It does give up a tiny bit of human-readable information, but it's more consistent and less coupled to the exact definition of `nominal_hash`.